app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2 class="actions">Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ Optional ] I think it would be better if you style the `Login` form the same way it is styled in the project wireframe. The Login button should be the same `width` as the `email input` and `password input` fields.

## Your design
![Screenshot from 2023-06-22 23-32-25](https://github.com/atok624/Blog-app-rails/assets/16392046/6e9fc059-57fc-40d9-a400-519c482c8867)


## Project wireframe
![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_login.png)